main/pipewire/template.py

```diff
@@ -9,7 +9,6 @@
     "-Dsdl2=disabled",  # examples
     "-Dsystemd=disabled",
     "-Dlibffado=disabled",
-    "-Droc=disabled",  # TODO
```

Comment:
should prolly split the module and the libs below and then install_if = parent, roc-toolkit

Reply:
should be good now